src/lib.rs

```diff
@@ -161,14 +168,22 @@ pub mod pallet {
 		/// The challenge duration is too large.
 		ChallengeDurationOverflow,
 
-		/// The channel is already concluded.
+		/// Operation is invalid in current phase.
+		WrongPhase,
```

Comment:
A more explicit name could be:
```suggestion
		DisputeAlreadyRegistered,
```
from looking at this line: https://github.com/perun-network/perun-polkadot-pallet/blob/f7823817297615472058670859381e113a8e3122/src/lib.rs#L305

Reply:
The `ensure` statement requires that, if a dispute has been registered already, that dispute must still be in phase `Register`. (If the dispute period is over and we progressed into one of the next phases, we can't refute the registered state anymore.)

`DisputeAlreadyRegistered` is not suitable here because it's OK if we already registered before, we just still need to be in the register phase (the phase where we can only dispute with fully signed states).

I will rename to `RegisterPhaseOver`. Does that make sense to you?

Reply:
I see. Thanks it makes sense 👍